.gitignore

```diff
@@ -1,3 +1,24 @@
-#Ignore files
-
-.DS_Store
```

Comment:
I have `.DS_Store` in my global gitignore. We can leave it here also I'll put it back